packages/vscode-extension/src/commonlib/cacheAccess.ts

```diff
@@ -8,11 +8,11 @@ import { TokenCacheContext } from "@azure/msal-node";
 import { ConfigFolderName } from "@microsoft/teamsfx-api";
 import * as crypto from "crypto";
 import * as fs from "fs-extra";
-import * as keytarType from "keytar";
 import VsCodeLogInstance from "./log";
 import * as os from "os";
-import { env } from "vscode";
 import { localize } from "../utils/localizeUtils";
+import { context } from "../globalVariables";
+import { SecretStorage } from "vscode";
```

Comment:
Any version requirement for vscode? E.g., the `engines:vscode` value (https://github.com/OfficeDev/TeamsFx/blob/dev/packages/vscode-extension/package.json#L24) in *package.json*?

Reply:
Thanks for the reminder. I noticed that the 'secretStorage' API was added around 2020, and since VS Code 1.66.0 was released in 2022, we may not need to add an additional version check.